health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -157,31 +169,26 @@ const Hypothesis = ({ category, assumptions: initialAssumptions,setShowToast,all
                   id={index}
                   placeholder={t("MP_ENTER_ASSUMPTION")}
                   onChange={(event) => {
-
                     handleAssumptionChange(category, event, item);
-
                   }}
                 />
                 <div className="hypothesis-delete-button">
                   <DeleteIconv2 />
-                  <span style={{ color: "#B91900", textDecoration: "Underline" }} onClick={() => handleDeleteClick(index)}>{t("DELETE")}</span>
+                  <span style={{ color: "#B91900", textDecoration: "Underline" }} onClick={() => handleDeleteClick(index)}>
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)